ros_installation.md

```diff
@@ -0,0 +1,197 @@
+version. For a fully-integrated ROS system, Citadel is the recommened
+version. The way to proceed is just to use the ROS repository and do ***not***
+use the osrfoundation repository.
+
+ROS packages `ros-noetic-ros-gz` will be available pulling the necessary
+Gazebo Citadel dependencies.
+
+## Using an specific Gazebo version with ROS2
+***Warning!:*** Using this option, you won't be able to use any ROS Ubuntu package
+related to Gz libraries from ROS deb repository.
+
+Selecting a different release of Gz than the one officially supported in each
+ROS distribution will require to compile the [`ros_gz
+bridge`](https://github.com/gazebosim/ros_gz) from source. Same applies for
+other ROS packages using Gz libraries that could also require code changes in
+they are not supporting the new Gz version chosen.
+
+### Gz Garden
+
+Gz Garden can be used with ROS2 Humble and ROS2 Rolling, the [`ros_gz
+bridge`](https://github.com/gazebosim/ros_gz) supports the compilations from
+source.
+
+ * Folow [the instruction to install gz-garden](https://gazebosim.org/docs/garden/install_ubuntu#binary-installation-on-ubuntu)
+   from osrfoundation.org repository
+ * Follow the instructions to compile the ros_gz bridge from source in a colcon workspace
+   * [ROS2 Humble](https://github.com/gazebosim/ros_gz/tree/humble#from-source)
+     * Be sure of using `export GZ_VERSION=humble`
+   * [ROS2 Rolling](https://github.com/gazebosim/ros_gz/tree/ros2#from-source)
+     * Be sure of using `export GZ_VERSION=rolling`
+
+### Gz Fortress
+
+Gz Fortress can be also used with ROS2 Galactic and ROS1 Noetic, the [`ros_gz
+bridge`](https://github.com/gazebosim/ros_gz) supports the compilation from
+source.
+
+ * Folow [the instruction to install gz-fortress](https://gazebosim.org/docs/fortress/install_ubuntu#binary-installation-on-ubuntu)
+   from osrfoundation.org repository
+ * Follow the instructions to compile the ros_gz bridge from source in a colcon workspace
+   * [ROS2 Galactic](https://github.com/gazebosim/ros_gz/tree/galactic#from-source)
+     * Be sure of using `export GZ_VERSION=fortress`
+   * [ROS1 Noetic]()https://github.com/gazebosim/ros_gz/tree/noetic#from-source
+     * Be sure of using `export GZ_VERSION=fortress`
+
+### Gz Edifice
+Gz Edifice can be also used with ROS2 Foxy, ROS2 Rolling and ROS1 Noetic, the [`ros_gz
+bridge`](https://github.com/gazebosim/ros_gz) supports the compilation from
+source.
+
+ * Folow [the instruction to install gz-edifice](https://gazebosim.org/docs/edifice/install_ubuntu#binary-installation-on-ubuntu)
+   from osrfoundation.org repository
+ * Follow the instructions to compile the ros_gz bridge from source in a colcon workspace
+   * [ROS2 Foxy](https://github.com/gazebosim/ros_gz/tree/foxy#from-source)
+     * Be sure of using `export GZ_VERSION=foxy`
+   * [ROS2 Rolling](https://github.com/gazebosim/ros_gz/tree/ros2#from-source)
+     * Be sure of using `export GZ_VERSION=rolling`
```

Comment:
This is wrong GZ_VERSION should't be `rolling`

Reply:
right, all them should be Edifice [b37e02e](https://github.com/gazebosim/docs/pull/339/commits/b37e02eddb80bc21d66b3cff4f3c9589f3812272)